src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LbfgsPredictorBase.cs

```diff
@@ -47,7 +47,7 @@ public abstract class OptionsBase : LearnerInputBaseWithWeight
             [Argument(ArgumentType.AtMostOnce, HelpText = "Maximum iterations.", ShortName = "maxiter")]
             [TGUI(Label = "Max Number of Iterations")]
             [TlcModule.SweepableLongParamAttribute("MaxIterations", 1, int.MaxValue)]
-            public int MaxIterations = Defaults.MaxIterations;
+            public int MaximumIterations = Defaults.MaximumIterations;
```

Comment:
>MaximumIterations [](start = 23, length = 17)

let's stick to NumberOfIterations across all our learners.

Reply:
fixed

---
In reply to: [260873873](https://github.com/dotnet/machinelearning/pull/2761#discussion_r260873873) [](ancestors = 260873873)